test/test_permeation_problem.py

```diff
@@ -152,8 +159,8 @@ def test_permeation_problem_multi_volume():
 
     times, flux_values = my_model.run()
 
-    # -------------------------- analytical solution -------------------------------------
-    D = my_mat.get_diffusion_coefficient(my_mesh.mesh, temperature)
+    # ---------------------- analytical solution -----------------------------
+    D = my_mat.get_diffusion_coefficient(my_mesh.mesh, temperature, my_model.species[0])
```

Comment:
Is this needed? the `species` argument is optional when there's only one species